notebooks/phish_detection/Phishing_Detection_using_Bert_CLX.ipynb

```diff
@@ -73,10 +73,32 @@
      "name": "stderr",
```

Comment:
<p>Same thing as CLAIR - let's say a few sentences about this dataset and link to it. I know we pull it down ourselves from S3, but we should provide a link to the actual dataset (same with the other datasets below).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/232/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>